src/engine/Util/Detector.ts

```diff
@@ -5,9 +5,117 @@ module ex {
     * Excalibur internal feature detection helper class
     */
    export class Detector {
-      
+
+      private _features: Object = null;
```

Comment:
Because this object is exposed publicly, it would be preferable to expose the type information to consumers. Can we create an interface to use for this?

```ts
export interface DetectedFeatures {
    canvas: boolean;
    // etc...
}

private _features: DetectedFeatures = null;
```